concourse/pipelines/container-image-build.yaml

```diff
@@ -0,0 +1,20 @@
+resources:
```

Comment:
does this merit its own top level pipeline? if so, we should name it more specifically

Reply:
We have a bunch of container image need to build with concourse, right? I imagine that we put all these job in this pipeline